### PR TITLE
Feature/boostershot coverage

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
Update of booster layout on vaccine page and actueel landelijk

### CODE REVIEW

**Review by VWSCoronaDashboard19 — APPROVED** [review]

File: packages/app/src/domain/vaccine/components/metadata.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { MarginBottomProps } from 'styled-system';
+import { useIntl } from '~/intl';
+import { Box } from '~/components/base';
+import { Text } from '~/components/typography';
+
+export type source = {
```
Comment (VWSCoronaDashboard22) [inline packages/app/src/domain/vaccine/components/metadata.tsx:6]: Please capitalize the first letter of a type

File: packages/app/src/domain/vaccine/components/metadata.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  text: string;
+  href: string;
```
Comment (VWSCoronaDashboard22) [inline packages/app/src/domain/vaccine/components/metadata.tsx:8]: Why does a source require an href when it is not used in this component?

File: packages/app/src/domain/vaccine/components/metadata.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+};
+
+export interface MetadataProps extends MarginBottomProps {
```
Comment (VWSCoronaDashboard22) [inline packages/app/src/domain/vaccine/components/metadata.tsx:11]: Why extend this with MarginBottomProps?

File: packages/app/src/domain/topical/mini-vaccination-coverage-tile.tsx [hunk 4]
```diff
@@ -44,6 +56,17 @@ export function MiniVaccinationCoverageTile(
           valueLabel={fullyVaccinatedPercentageLabel}
           barLabel={fullyVaccinatedBarLabel}
         />
+        {vaccinationsBoosterCoverageFeature.isEnabled &&
+          boosterShotAdministered &&
+          boosterShotAdministeredBarLabel &&
+          boosterShotAdministeredLabel && (
```
Comment (VWSCoronaDashboard22) [inline packages/app/src/domain/topical/mini-vaccination-coverage-tile.tsx:62]: This value is not being passed in index.tsx and is undefined. Therefore the bar will never be visisble.

File: packages/app/src/components/metadata.tsx [hunk 5]
```diff
@@ -79,6 +87,7 @@ export function Metadata({
                       date: formatDateFromSeconds(obtainedAt, 'weekday-medium'),
                     }
                   )}`}
+                {`. ${intervalString}`}
```
Comment (VWSCoronaDashboard22) [inline packages/app/src/components/metadata.tsx:90]: Render conditionally. Value could be undefined

File: packages/app/src/domain/topical/mini-vaccination-coverage-tile.tsx [hunk 6]
```diff
@@ -44,6 +56,16 @@ export function MiniVaccinationCoverageTile(
           valueLabel={fullyVaccinatedPercentageLabel}
           barLabel={fullyVaccinatedBarLabel}
         />
+        {vaccinationsBoosterCoverageFeature.isEnabled &&
+          boosterShotAdministered &&
+          boosterShotAdministeredBarLabel && (
```
Comment (VWSCoronaDashboard20) [inline packages/app/src/domain/topical/mini-vaccination-coverage-tile.tsx:61]: Is there a particular reason for all these checks? (Except for the feature flag, I understand that one)

File: packages/app/src/domain/vaccine/vaccinations-booster-kpi-section.tsx [hunk 7]
```diff
@@ -54,75 +36,31 @@ export function VaccinationsBoosterKpiSection({
...
+              fontVariantNumeric: 'tabular-nums',
+              lineHeight: 1,
             })}
+          >
+            {formatNumber(dataBoosterShotPlanned)}
+          </Text>
+          <Markdown content={text.booster_planned_7_days.description} />
+          <LokalizeMetadata
+            date={text.booster_last_7_days.metadata_date}
+            source={source}
```
Comment (VWSCoronaDashboard20) [inline packages/app/src/domain/vaccine/vaccinations-booster-kpi-section.tsx:63]: The source doesn't work on the page. I will elaborate on it when I cover the vaccine page.

File: packages/app/src/domain/vaccine/vaccine-booster-administrations-kpi-section.tsx [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+                  <InlineText color="data.primary">
+                    {`${formatPercentageAsNumber(
+                      totalBoosterAndThirdShots.percentage
+                    )}%`}
+                  </InlineText>
+                ),
+              }
+            )}
+          </Text>
+          <Message variant="warning">
```
Comment (VWSCoronaDashboard20) [inline packages/app/src/domain/vaccine/vaccine-booster-administrations-kpi-section.tsx:88]: Shouldn't this check if the warning is set and display it based on that?

File: packages/app/src/pages/landelijk/vaccinaties.tsx [hunk 9]
```diff
@@ -511,75 +514,113 @@ const VaccinationPage = (props: StaticProps<typeof getStaticProps>) => {
...
                   data.booster_shot_administered.last_value
+                    .ggd_administered_last_7_days
                 }
-                dataBoosterShotPlanned={data.booster_shot_planned.last_value}
-                dataThirdShotAdministered={
-                  data.third_shot_administered.last_value
+                dataBoosterShotPlanned={
+                  data.booster_shot_planned.last_value.planned_7_days
                 }
+                source={text.vaccination_grade_toggle_tile.source}
```
Comment (VWSCoronaDashboard20) [inline packages/app/src/pages/landelijk/vaccinaties.tsx:599]: ```vaccination_grade_toggle_tile``` doesn't in its place here. Could that be why TypeScript is complaining, and the source for this component is curently displaying [object Object]?

**Review by VWSCoronaDashboard20 — CHANGES_REQUESTED** [review]

File: packages/app/src/domain/topical/mini-vaccination-coverage-tile.tsx (hunk 6 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/domain/topical/mini-vaccination-coverage-tile.tsx]: They are optional props that's why the check is there.

Comment (VWSCoronaDashboard20) [reply packages/app/src/domain/topical/mini-vaccination-coverage-tile.tsx]: Check! Is there a reason for it being optional? The reason for asking is looking at, for example, to the ```fullyVaccinatedPercentage``` there it is not.

**Pull request merged**